.travis.yml

```
@@ -8,19 +8,19 @@ addons:
   postgresql: '10'
 cache:
   directories:
-    - "$GOPATH/pkg/dep"
     - "$HOME/gopath/pkg/mod"
 env:
   global:
     - secure: KONX7893Jxeu3zBnNRWMl0x9DNsriN+RFojuFE28bDBjE2NJKTBeIhWNF4Jnoy7ba9hvE3LB7aGFUETnUR2sxKgNp/yz1Q6Wjk2qFvIPxYB8soOXWnOcRRvRulM2jOpLLJdT4aKewyTGTQfszib/3BD2fWgIgnH0aAvhoV6F/e9Mhtt1ZXLvvRDyDX2q0XkDtjK9tTB8/QOJFz6c9UI/fScYglI+Ob5lx7GwjYg6LHbIaPXt4wrGLXW9/FXOHTI1F6+FuDE08n7EtqkuEqmh3gURQRRyjmX2nDnxUuvSPpbTR6cRHctgwaYSL2gu0321lbXIUv4bFAW14MoQLnI5xKW8nFjuD0Z3DrePr3G96sTombrg2PeI0eM/twW9YgTlHpjt+gkblSriOZ8vVe1UDMZ0OidXSzO9dOk0vDustwvbA+TYP3/e7Tw60wn1NWoEFl38W8F+jB2kBuMON45AT4I5zPQAjZLmEwTgYm4jOahpnvOY3Z6cxqpN+zDNlqllETJCdf+e5bp8xk9vYZtEZ0ML3L7cMMw3IbLg+slE4rIUZuydvk4rC9EXwqPw/089BhCFhpc62GfvlVhDrPJfnHIPhzZ9nyYS1FDBl96gVNze2uFOyaEMACFtevFF4EP0NmUYRhLWPcasdjbhUjUsIiFQVS3CFQODR9FL0SXSM9M=
     - POSTGRES_DSN="postgres://postgres@localhost:5432/goengine?sslmode=disable&client_encoding=UTF8"
+    - GO111MODULE=on
 
 matrix:
   include:
     - name: "GoLang CI lint"
       go: 1.11.x
       install:
-        - dep ensure -v
+        - go mod vendor
```

```suggestion
        - go mod vendor -v
```

Do we want to enable verbose mode?:)